libs/sdk-ui-kit/src/Header/HeaderHelp.tsx

```diff
@@ -0,0 +1,115 @@
+// (C) 2021 GoodData Corporation
+import React, { useState } from "react";
+import { FormattedMessage, injectIntl, IntlShape } from "react-intl";
+import { Overlay } from "../Overlay";
+import cx from "classnames";
+import isEmpty from "lodash/isEmpty";
+
+interface itemPropType {
```

Comment:
Interface name has wrong shape, should be `IItemPropType`. Also, would `IHelpItem` be better?